File: Example/CirrusMDSDK/SecondViewController.m
//
//  SecondViewController.m
//  CirrusMDSDK
//
//  Created by CirrusMD.
//  Copyright (c) 2017 CirrusMD. All rights reserved.
//

#import "SecondViewController.h"
#import "ExampleViewController.h"

@interface SecondViewController ()

@end

@implementation SecondViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    // Do any additional setup after loading the view, typically from a nib.
}


- (void)didReceiveMemoryWarning {
    [super didReceiveMemoryWarning];
    // Dispose of any resources that can be recreated.
}

- (IBAction)showSDK:(id)sender {
    UIViewController* example = [[ExampleViewController alloc] init];
    [self presentViewController:example animated:YES completion:^{}];
}

@end

File: CirrusMDSDK-Pods/SecondViewController.swift
//
//  SecondViewController.swift
//  CirrusMDSDK-Swift-Pods
//
//  Created by Taylor Case on 9/13/19.
//  Copyright © 2019 steam. All rights reserved.
//

import UIKit

class SecondViewController: UIViewController {
    
    @IBAction func showSdk(_ sender: Any) {
        let example = ExampleViewController()
        present(example, animated: true, completion: nil)
    }
    
}

